tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -115,6 +115,18 @@ Search and Verify GPU Items Appears In Resources Page
     Search Items In Resources Section   GPU
     Check GPU Resources
 
+Verify Failing Images Are Shown In Notifications
+    [Documentation]    Verifies failing Notebook names are shown in RHODS Dashboard Notifications
+    [Tags]    Sanity
+    ...       ODS-470
+    Clear RHODS Notifications
+    ${failed_build_name} =    Provoke Image Build Failure    namespace=redhat-ods-applications
+    ...    build_name_includes=pytorch    build_config_name=s2i-pytorch-gpu-cuda-11.4.2-notebook
+    ...    container_to_kill=sti-build
+    RHODS Notification Drawer Should Contain  message=Notebook image build PyTorch failed
+    Delete Failed Build And Start New One  namespace=redhat-ods-applications  failed_build_name=${failed_build_name}  build_config_name=s2i-pytorch-gpu-cuda-11.4.2-notebook
```

Comment:
this should be part of the teardown. If the previous line fails, this tc would leave the build broken

Reply:
Updated.